core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -407,6 +407,24 @@ public ServerBuilder service(PathMapping pathMapping, Service<HttpRequest, HttpR
     }
 
     /**
+     * Decorates and binds the specified {@link ServiceWithPathMappings} at multiple {@link PathMapping}
+     * of the default {@link VirtualHost}.
```

Comment:
Since we don't have a version without a decorator should we mention something like

Use Function.identity when decoration is not needed.